templates/aboutUsSocialLinks.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export default function AboutUsSocialLinks (props) {
+  const {
+    socialLinks
+  } = props
+
+  return (
+    <div className='aboutus__sociallinks' role='list'>
+      {socialLinks.map((socialLink) => {
+        if (socialLink._link){
```

Comment:
Minor: Can you use a ternary expression here instead of the 'if' wrapper
```suggestion
        !socialLink._link ? '' : <>return (
            <a href={socialLink._link} className={`icon icon-${socialLink._service?.toLowerCase()}`}
              target='_blank' title={`Follow us on ${socialLink._service}`} aria-label={`Follow us on ${socialLink._service}`}>
            </a>
```

Reply:
I can try to. Why? Personally this would read as confusing and I try to make my code as easy to understand as possible

Reply:
I just mention it for consistency with the work on other templates. Typically we haven't went with if blocks but used https://legacy.reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator